ci/python-publish.yml

```
@@ -0,0 +1,27 @@
+name: Upload Python Package
+
+on: [release]
```

```suggestion
on:
  release:
    types: [created]
```
This sill make sure this workflow only runs when the release is created and not on any of the other events. Then remove all of the conditionals on the steps.

---

Thanks! I was looking for a better way to do this but didn't notice this in the actions documentation.